receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -952,3 +1046,147 @@ func TestGetCommitInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestGetRepoCVEs(t *testing.T) {
+	testCases := []struct {
+		desc             string
+		server           *http.ServeMux
+		expectedErr      error
+		expectedCVECount int
+	}{
+		{
+			desc: "TestSinglePageResponse",
+			server: MockServer(&responses{
+				scrape: false,
+				cveResponse: cveResponse{
+					cves: []VulnerabilityAlerts{
+						{
+							Nodes: []CVENode{
+								{
+									SecurityVulnerability: CVENodeSecurityVulnerability{
+										Severity: "HIGH",
+									},
+								},
+								{
+									SecurityVulnerability: CVENodeSecurityVulnerability{
+										Severity: "MODERATE",
+									},
+								},
+							},
+						},
+					},
+					responseCode: http.StatusOK,
+				},
+			}),
+			expectedErr:      nil,
+			expectedCVECount: 2,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.desc, func(t *testing.T) {
+			factory := Factory{}
+			defaultConfig := factory.CreateDefaultConfig()
+			settings := receivertest.NewNopCreateSettings()
+			ghs := newGitHubScraper(context.Background(), settings, defaultConfig.(*Config))
+			server := httptest.NewServer(tc.server)
+			defer server.Close()
+			client := graphql.NewClient(server.URL, ghs.client)
+
+			cves, err := ghs.getCVEs(context.Background(), client, "repo1")
+
+			assert.Equal(t, tc.expectedCVECount, len(cves.Repository.VulnerabilityAlerts.Nodes))
+			if tc.expectedErr == nil {
+				assert.NoError(t, err)
+			} else {
+				assert.EqualError(t, err, tc.expectedErr.Error())
+			}
+		})
+	}
+}
+
+func TestAggregateSeverity(t *testing.T) {
```

Comment:
```suggestion
func TestMapSeverities(t *testing.T) {
```

This test function name should be `Test<MyFunc>` and the function name changes right?

Reply:
The function name has been updated. Marking as resolved